src/backend/libs/efiling-lookup-client/src/test/java/ca/bc/gov/open/jag/efilinglookupclient/EfilingLookupClientTest.java

```diff
@@ -0,0 +1,17 @@
+package ca.bc.gov.open.jag.efilinglookupclient;
+
+import org.junit.jupiter.api.*;
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("Efiling Lookup Client Test Suite")
+public class EfilingLookupClientTest {
+
+
+    @DisplayName("CASE 1:")
+    @Test
+    public void callLookupTest() {
+
```

Comment:
are we ok with merging an empty test suite?

Reply:
Right now this is an empty service, no functions implemented.